setup.py

```diff
@@ -17,7 +17,7 @@
         'cryptography',
     ],
     extras_require={
-        "dev": ["nose", "tox"],
+        "dev": ["nose", "tox", "mypy == 0.910"],
```

Comment:
Maybe only freeze major version?

Reply:
Pinning the version was recommended here:
https://mypy.readthedocs.io/en/stable/existing_code.html#continuous-integration

I guess the reasoning is, you don't want CI to break unexpectedly if mypy releases a new minor version that emits warnings in different cases.